Sources/Commands/init.swift

```diff
@@ -30,10 +30,25 @@ extension InitError: CustomStringConvertible {
     }
 }
 
+/// Create an initial template package.
 final class InitPackage {
+    let rootd = POSIX.getcwd()
+
+    /// The mode in use.
     let mode: InitMode
+
+    /// The name of the example package to create.
     let pkgname: String
```

Comment:
This might be too picky to even mention, but shouldn't this use camel case like the other properties?  i.e. shouldn't it be "packageName"?